resources/todomvc/vanilla-examples/javascript-web-components/src/components/todo-app/todo-app.component.js

```diff
@@ -1,11 +1,9 @@
 import template from "./todo-app.template.js";
 import { useRouter } from "../../hooks/useRouter.js";
 
-// [TO-D0]: use local package for styles when available:
-// https://github.com/WebKit/Speedometer/pull/254
-import globalStyles from "../../styles/global.constructable.js";
-import appStyles from "../../styles/app.constructable.js";
-import mainStyles from "../../styles/main.constructable.js";
+import globalStyles from "../../../node_modules/todomvc-css/dist/global.constructable.js";
+import appStyles from "../../../node_modules/todomvc-css/dist/app.constructable.js";
+import mainStyles from "../../../node_modules/todomvc-css/dist/main.constructable.js";
```

Comment:
These seem wrong?? Why are we referring to files inside node_modules?

Reply:
since this is a native javascript module, I can't use "bare" imports, since [bare imports are not supported](https://jakearchibald.com/2017/es-modules-in-browsers/#bare-import-specifiers-arent-currently-supported).

Reply:
something like `import appStyles from "todomvc-css/dist/app.constructable.js";` does not work.